src/types/onyx/Report.ts

```diff
@@ -101,6 +104,9 @@ type Report = {
     /** The report visibility */
     visibility?: ValueOf<typeof CONST.REPORT.VISIBILITY>;
 
+    /** Whether the report has outstanding IOU */
+    hasOutstandingIOU?: boolean;
```

Comment:
We explicitly removed this property recently - https://github.com/Expensify/App/pull/32653. So, I am not sure we should back it again.

Reply:
It exists in the current main, since I do just migration and don't change the logic, I'm not sure if I should remove it
https://github.com/Expensify/App/blob/5b03db96e5a2816ba18cd5757f076037807baa90/src/libs/actions/IOU.js#L3448-L3454
https://github.com/Expensify/App/blob/5b03db96e5a2816ba18cd5757f076037807baa90/src/libs/actions/IOU.js#L3405-L3410

Reply:
Got it. @mountiny, FYI.

Reply:
will remove

Reply:
https://github.com/Expensify/App/pull/34593

Reply:
@VickyStash, so you can do not add `hasOutstandingIOU`.

Reply:
Merged to main so you can just sync up